### PR TITLE
Update types and add functionality

### PR DESCRIPTION
**Pull request by drbh**
updated instrument and added an options position structure

new functions
- get all past orders
- get open equity positions
- get open options positions
- get level II price book data (with Robinhood gold)

### CODE REVIEW

File: positions.go [hunk 1]
```diff
@@ -14,9 +14,45 @@ type Position struct {
...
+
 // GetPositions returns all the positions associated with an account.
-func (c *Client) GetPositions(a Account) ([]Position, error) {
-	return c.GetPositionsParams(a, PositionParams{})
+func (c *Client) GetOptionPositions() ([]OptionPostion, error) {
+	return c.GetOptionPositionsParams(PositionParams{NonZero: true})
+}
+
+// GetPositions returns all the positions associated with an account.
+func (c *Client) GetPositions() ([]Position, error) {
```
Comment (andrewstuart) [inline positions.go:54]: This looks like a breaking API change. Is there a specific reason to pull the Account out of the parameters?

Comment (drbh) [reply positions.go:54]: It looks like Robinhood used to return a string url on the account but now you can just call an endpoint for "positions" and Robinhood know who the caller is from the auth header. 

here is where that url was used 
https://github.com/andrewstuart/go-robinhood/blob/463e091050a8464f2b9ac41db3ad7182649dd655/positions.go#L41

but that returns blank and results in the method not returning anything.

TLDR; The Account param isnt needed and is not used
